### ISSUE TITLE
About the weight of classes.

### ISSUE THREAD

**Issue by tfzhang006** [xinge008/Cylinder3D#6]
How do you determine the weight of classes in CE loss? Thx.

**Comment by xinge008** [comment]
we count the point of each category in the training set, and take the reciprocal as the weights

**Comment by ouenal** [comment]
Looking at the loss_builder, unweighted cross entropy is returned. Where are the weights set exactly?
